Dockerfile

```diff
@@ -1,4 +1,35 @@
-FROM docker-registry.ripe.net/swe/gitlab-ci/awsebcli:3.18.2
+FROM mozilla/sbt as build
```

Comment:
You don't specifically need an `sbt` image as build, since you only use it to copy files into the distro-less image. Maybe use a less-specific container as `alpine:latest`.

Reply:
That is the container I started with before we decided to make this Dockerfile depend on our CI/CD workflow. It also enables the fallback build that is described in the Dockerfile.

In hindsight I'm not that happy with that change. Would you be ok with building in the multi-step container again?

Reply:
I'm just sticking with this container for now. I'll reconsider building in the container once the PG stuff is done.